src/Player/HackerStateConst.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+
+struct HackerStateConst {
```

Comment:
Where did you get those values?

Reply:
`HackerStateConst::HackerStateConst`. Will add the ctor at the bottom of the struct.

Reply:
Can you add its code as well? I'm not quite sure if this should be a `struct` or `class`.